ts/Core/Series/Point.ts

```diff
@@ -122,6 +122,7 @@ class Point {
     public formatPrefix: string = 'point';
     public graphic?: SVGElement;
     public graphics?: Array<SVGElement|undefined>;
+    public hiddenInDataClass?: boolean = false;
```

Comment:
```suggestion
    public hiddenInDataClass?: boolean;
```
If we declare it but not set it to false, it will still work as intended as a truthy check, but I believe we will save us all of the extra bytes. If not, we can define it just in the PointLike type if I remember correctly, so it becomes a pure TS abstraction.